google-cloud-clients/google-cloud-bigquery/README.md

```diff
@@ -19,7 +19,7 @@ If you are using Maven with a BOM, add this to your pom.xml file.
       <dependency>
         <groupId>com.google.cloud</groupId>
         <artifactId>libraries-bom</artifactId>
-        <version>2.8.0</version>
+        <version>2.9.0</version>
```

Comment:
[java-compute](https://github.com/googleapis/java-compute),
[java-container](https://github.com/googleapis/java-container),
[java-containeranalysis](https://github.com/googleapis/java-containeranalysis),
[java-talent](https://github.com/googleapis/java-talent),
[java-tasks](https://github.com/googleapis/java-tasks),
[java-asset](https://github.com/googleapis/java-asset),
[java-logging](https://github.com/googleapis/java-logging),
[java-datalabeling](https://github.com/googleapis/java-datalabeling),
[java-datacatalog](https://github.com/googleapis/java-datacatalog),
[java-recaptchaenterprise](https://github.com/googleapis/java-recaptchaenterprise),
[java-speech](https://github.com/googleapis/java-speech),
[java-texttospeech](https://github.com/googleapis/java-texttospeech),
[java-trace](https://github.com/googleapis/java-trace),
[java-translate](https://github.com/googleapis/java-translate),
[java-vision](https://github.com/googleapis/java-vision),
[java-video-intelligence](https://github.com/googleapis/java-video-intelligence),
[java-webrisk](https://github.com/googleapis/java-webrisk),
[java-websecurityscanner](https://github.com/googleapis/java-websecurityscanner),
[java-automl](https://github.com/googleapis/java-automl),
[java-dataproc](https://github.com/googleapis/java-dataproc),
[java-dialogflow](https://github.com/googleapis/java-dialogflow),
[java-dlp](https://github.com/googleapis/java-dlp),
[java-dns](https://github.com/googleapis/java-dns),
[java-scheduler](https://github.com/googleapis/java-scheduler),
[java-errorreporting](https://github.com/googleapis/java-errorreporting),
[java-gameservices](https://github.com/googleapis/java-gameservices),
[java-resourcemanager](https://github.com/googleapis/java-resourcemanager),
[java-redis](https://github.com/googleapis/java-redis),
[java-recaptchaenterprise](https://github.com/googleapis/java-recaptchaenterprise),
[java-phishingprotection](https://github.com/googleapis/java-phishingprotection),
[java-monitoring](https://github.com/googleapis/java-monitoring),
[java-language](https://github.com/googleapis/java-language),
[java-kms](https://github.com/googleapis/java-kms),
[java-iamcredentials](https://github.com/googleapis/java-iamcredentials)

still need to add same changes in above clietns repo as well.

Reply:
Yes, but that doesn't have to happen in this PR.

Reply:
yes,that's correct.Can you please raise PR for other client as well.